### ISSUE TITLE
Save field

### ISSUE THREAD

**Issue by wateengoedidee** [fruitstudios/Icons#9]
Hi Fruitstudio's,

When I create a new field in craft and select icon (fruiticons), and choose ionicons as font, is saves the field with Font Awesome selected. When I edit the field and choose an other font, it keeps saving FA as the chosen font.

Bug?

**Comment by samhibberd** [comment]
Yeah sorry, we do have a new version but haven't released it yet. Any particular icon font you needed?

**Comment by wateengoedidee** [comment]
Hi Sam,

No problem, we need the ionicons font. Can we use/fork the new version to
try?



Met vriendelijke groet,

Tom Fischer

Bezoekadres Het Eeftink 11-21
7541 WH Enschede
T +316 24 89 41 19
E info@wateengoedidee.nl
W www.wateengoedidee.nl

Op 1 juni 2016 bij 12:26:55, Sam Hibberd (notifications@github.com) schreef:

Yeah sorry, we do have a new version but haven't released it yet. Any
particular icon font you needed?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/fruitstudios/Icons/issues/9#issuecomment-222953647>,
or mute
the thread
<https://github.com/notifications/unsubscribe/ACOOU3SY_nOSBYftPyFkNM-T5cLfstu7ks5qHV5vgaJpZM4Iqekx>
.

**Comment by wateengoedidee** [comment]
Hi Sam,

No problem, we need the ionicons font. Can we use/fork the new version to try?

**Comment by samhibberd** [comment]
I have emailed you directly, did you receive??

**Comment by wateengoedidee** [comment]
Hi Sam,

I received your e-mail. It went directly to my spam folder, so that’s why I
didn’t saw it ;)

I’m going to test it, thanks in advance.



Met vriendelijke groet,

Tom Fischer

Bezoekadres Het Eeftink 11-21
7541 WH Enschede
T +316 24 89 41 19
E info@wateengoedidee.nl
W www.wateengoedidee.nl

Op 1 juni 2016 bij 15:35:15, Sam Hibberd (notifications@github.com) schreef:

I have emailed you directly, did you receive??

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/fruitstudios/Icons/issues/9#issuecomment-222993689>,
or mute
the thread
<https://github.com/notifications/unsubscribe/ACOOU-XkwNAj7ZnCH-VMJYCiPWdghzbMks5qHYqTgaJpZM4Iqekx>
.

**Comment by wateengoedidee** [comment]
When I install the plugin, and make a field, the icons are not loaded.. Did
you sent me a working version? ;)

In the config I see fontawesome, and in the vendors it’s called
font-awesome, maybe some issues?




Met vriendelijke groet,

Tom Fischer

Bezoekadres Het Eeftink 11-21
7541 WH Enschede
T +316 24 89 41 19
E info@wateengoedidee.nl
W www.wateengoedidee.nl

Op 1 juni 2016 bij 16:00:29, Wat een goed idee (info@wateengoedidee.nl)
schreef:

Hi Sam,

I received your e-mail. It went directly to my spam folder, so that’s why I
didn’t saw it ;)

I’m going to test it, thanks in advance.



Met vriendelijke groet,

Tom Fischer

Bezoekadres Het Eeftink 11-21
7541 WH Enschede
T +316 24 89 41 19
E info@wateengoedidee.nl
W www.wateengoedidee.nl

Op 1 juni 2016 bij 15:35:15, Sam Hibberd (notifications@github.com) schreef:

I have emailed you directly, did you receive??

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/fruitstudios/Icons/issues/9#issuecomment-222993689>,
or mute
the thread
<https://github.com/notifications/unsubscribe/ACOOU-XkwNAj7ZnCH-VMJYCiPWdghzbMks5qHYqTgaJpZM4Iqekx>
.
